db/sql/test_sql_functions.sql

```diff
@@ -298,6 +296,103 @@ END;
 $f$ LANGUAGE plpgsql;
 
 
+
+CREATE OR REPLACE FUNCTION __setup_msar_prepare_temp_table() RETURNS SETOF TEXT AS $$
+BEGIN
+  DROP TABLE IF EXISTS tmp_src_prepare;
+  CREATE TABLE tmp_src_prepare(id text, other text);
+  INSERT INTO tmp_src_prepare VALUES ('1','a'), ('2','b');
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION test_msar_prepare_temp_table_for_import() RETURNS SETOF TEXT AS $f$
+DECLARE
+  r_json jsonb;
+  table_oid bigint;
+BEGIN
+  PERFORM __setup_msar_prepare_temp_table();
+
+  -- Test with generated table name
+  r_json := msar.prepare_temp_table_for_import(
+    tab_name := 'tmp_test_' || replace(clock_timestamp()::text, ' ', '_'),
```

Comment:
Is there a specific reason why you decided to use `timestamps` for table names? You can use a regular name, the tables created exist only till the test is completed, once the test is complete, the tables are deleted.